### PR TITLE
Feature/sonya

### PR DESCRIPTION
**Pull request by sonyashijin**
# *Name of the PR*

## :recycle: Current situation & Problem
Not added to class app

## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ ] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: CS342MemberViews/Sources/SonyaJin/ProfileView.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+// SPDX-License-Identifier: MIT
+//
+import SwiftUI
+
+
+public struct ProfileView: View {
+    public var body: some View {
+        profileImage
+            .resizable()
+            .aspectRatio(1.0, contentMode: .fit)
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/SonyaJin/ProfileView.swift]: You can add the `scaledToFill()` modifier to fix the squeezing of your image:
```suggestion
            .resizable()
            .scaledToFill()
            .aspectRatio(1.0, contentMode: .fit)
```

File: CS342MemberViews/Sources/SonyaJin/ProfileView.swift [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            .clipShape(Circle())
+            .shadow(radius: 10)
+            .padding(.horizontal, 32)
+            .frame(maxWidth: 300)
+    }
+    
+    private var profileImage: Image {
+        guard let imagePath = Bundle.module.path(forResource: "SonyaJin", ofType: "jpeg"),
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/SonyaJin/ProfileView.swift]: Your image is not displayed in the view. This is due to the fact that you have added the wrong path extension:
```suggestion
        guard let imagePath = Bundle.module.path(forResource: "SonyaJin", ofType: "jpg"),
```

File: CS342MemberViews/Sources/SonyaJin/Resources/SonyaJinBio.md [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/SonyaJin/Resources/SonyaJinBio.md]: You Bio does not include any information.

File: CS342MemberViews/Package.swift [hunk 4]
```diff
@@ -63,6 +64,19 @@ let package = Package(
...
+        ),
+        .target(
+            name: "SonyaJin",
+            dependencies: [
+                .target(name: "InstructorViews")
+            ],
+            exclude: [
+                "Resources/SonyaJin.jpeg.license",
+                "Resources/SonyaJin.md.license"
+            ],
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Package.swift]: You get an error message that you having an invalid `exclude` statement. 
You are trying to exclude the _"Resources/SonyaJin.jpeg.license"_ and _"Resources/SonyaJin.md.license"_ files but you have not added any of these files to the repo. Please add these files including a proper license so your REUSE checks are passing. You can find out more about the REUSE and SwiftLint checks here: https://reuse.software/spec/ . Please refer to the other Swift Package targets and files to see the typical license header that we are using.

**Review by PSchmiedmayer — CHANGES_REQUESTED** [review]
Hi @sonyashijin,

Thank you for your PR and for adding some information about you to the CS342 class application!

I have added some comments to your current state of the PR on improving your code and adhering to the REUSE specification.

Here are the next steps to successfully submit your PR:
1. One of the tasks of Assignment 1 is a creative and nice addition beyond the views that we have in the application. Feel free to add a more creative solution for the second round of review if you want to. You don't have to add any additional views if you don't want to, but you will miss out on some bonus points for Assignment 1, but can still get a 100% if you fulfill all other requirements including solving all SwiftLint errors, the REUSE specification, and adding UI tests.
2. As already noted, you are also missing any UI tests to verify your functionality. As described in Assignment 1, you are required to add at least one basic UI test.
3. Please address my comments in the PR that you can see below this message.
4. Improve your PR description above to detail your changes and remove sections that are not applicable.

I have also merged the latest main state in your branch to make merging the PR easy so you can see the current checks that are failing. You will have to at least solve one merge conflict before you merge the PR, as there will likely be some additional commits to the main branch until then.
Be sure that you `git pull` the latest state to your local version if you accept any of the suggestions here on GitHub and get the merge conflict to your local branch 👍

Please reach out if you have any questions or comments regarding the review instructions.

File: CS342MemberViews/Sources/SonyaJin/ProfileView.swift (hunk 2 above)
Comment (PSchmiedmayer) [reply CS342MemberViews/Sources/SonyaJin/ProfileView.swift]: @sonyashijin I have committed a fix to your issue here to get you started with adding some more creative views.

File: CS342MemberViews/Sources/SonyaJin/ProfileView.swift (hunk 1 above)
Comment (PSchmiedmayer) [reply CS342MemberViews/Sources/SonyaJin/ProfileView.swift]: @sonyashijin I have committed a fix to your issue here to get you started with adding some more creative views.

**Comment by vishnuravi** [comment]
@PSchmiedmayer Sonya and I met and worked on these revisions together. Please re-review and let her know if you have any other comments!

**Review by PSchmiedmayer — APPROVED** [review]

**Comment by PSchmiedmayer** [comment]
@sonyashijin Thank you for your changes. All tests passed and I merged your changes 🚀

**Comment by sonyashijin** [comment]
@PSchmiedmayer @vishnuravi Thank you for reviewing and for your help!
